src/libs/kata-types/src/config/hypervisor/mod.rs

```diff
@@ -1044,6 +1044,18 @@ impl SharedFsInfo {
     }
 }
 
+/// Configuration information for remote hypervisor type.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct RemoteInfo {
+    /// Remote hypervisor socket path
+    #[serde(default)]
+    pub hypervisor_socket: String,
+
+    /// Remote hyperisor timeout of creating (in seconds)
+    #[serde(default)]
+    pub hypervisor_timeout: i32,
+}
+
```

Comment:
It seems that this hunk and the corresponding users belong to some other commit, maybe the previous one.

Reply:
I understand the motivation behind this as those two keys only pertain to the remote hypervisor.  However it's rather inconsistent with how the whole config info is stored.  There are many other keys that only pertain to a subset of hypervisors, or even a single one, and they are still direct members of their config structures for better or worse.

Arguably these two keys should be members of perhaps `BootInfo` since they seem to be what's need to "boot up" the remote hypervisor.